GATK/4.1.3.0/BaseRecalibration.nf

```diff
@@ -0,0 +1,26 @@
+
+
+process BaseRecalibration {
+    tag {"BaseRecalibration ${sample_id}.${int_tag}"}
+    label 'GATK'

```

Comment:
Should we add Labels for all processes?

Reply:
And maybe standardize tag names as well? For example 'Tool_SubTool_Sample'

Reply:
I think having a label with the toolname and version of said tool would be good.
Also agree on the tag names (don't really like the underscores though), so for bwa this would be:
```suggestion
    tag {"BWA mem ${sample_id} ${interval_name}"}
```
The interval part of course being optional